dist/plugins/DefaultPlugin/resources/configurationFiles/default.xml

```diff
@@ -39,9 +39,9 @@ up and is included in the configuration. The configuration settings directly ove
     </enumerations>
     <processingTools>
         <tool name='compressionDetection' value='determineFileCompressor.sh' basepath='roddyTools'/>
-        <tool name='createLockFiles' value='createLockFiles.sh' basepath='roddyTools'/>
-        <tool name='streamBuffer' value='streamBuffer.sh' basepath='roddyTools'/>
+        <!--<tool name='createLockFiles' value='createLockFiles.sh' basepath='roddyTools'/>-->
```

Comment:
They are not there anymore. Together with the new checks, Roddy would not start. So I removed them.